### PR TITLE
Add --node-selector flag to operator-k8s-plugin/install.sh for openshift.

### PR DESCRIPTION
**Pull request by pure-jliao**
for CI testing purpose, the `--node-selector` should be empty in the CI test

### CODE REVIEW

**Comment by pure-jliao** [comment]
@sdodsley - `--node-selector==""` is required for our internal functional tests, but customer may not want it due to security reason. So what should be the most common case for this? Does this change make sense, or we should use the default `node-selector` by default, only set it to empty in our CI test?

File: operator-k8s-plugin/install.sh [hunk 1]
```diff
@@ -3,10 +3,11 @@ IMAGE=quay.io/purestorage/pso-operator:v0.0.11
 NAMESPACE=pso-operator
 KUBECTL=oc
 ORCHESTRATOR=k8s
+NODESELECTOR=""
 
 usage()
 {
-    echo "Usage : $0 --image=<imagename> --namespace=<namespace> --orchestrator=<orchestrator> -f <values.yaml>"
+    echo "Usage : $0 --image=<imagename> --namespace=<namespace> --orchestrator=<orchestrator> --node-selector=<node-selector> -f <values.yaml>"
```
Comment (dsupure) [inline operator-k8s-plugin/install.sh]: node-selector is only used in openshift. This message will confuse k8s users.

Comment (pure-jliao) [reply operator-k8s-plugin/install.sh]: yes, simply removed this.

**Comment by pure-jliao** [comment]
> We should not mess with the default `node-selector` setting. This will go against the prefered OpenShift standard.
> We should keep the current docs comment about how to allow PSO to run on master and infra nodes however.
> I would say only set this to be blank for our internal testing CI suite.

Make sense, updated accordingly.

File: operator-k8s-plugin/install.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@ IMAGE=quay.io/purestorage/pso-operator:v0.0.11
 NAMESPACE=pso-operator
 KUBECTL=oc
 ORCHESTRATOR=k8s
+NODESELECTOR="default"
```
Comment (dsupure) [inline operator-k8s-plugin/install.sh]: Is this the default value used in the openshfit? Why don't we use a more unique string like "unset" ?

Comment (pure-jliao) [reply operator-k8s-plugin/install.sh]: not used, just an indicator for whether we should use an external parameter. yep `unset` might be better.

File: operator-k8s-plugin/README.md [hunk 3]
```diff
@@ -56,11 +56,9 @@ The install script will do the following:
 1. Create New Project.<br/>
 The script creates a new project (if it does not already exist) with the given namespace. If no namespace parameter is specified, the ``pso-operator`` namespace is used.<br/> 
 **OpenShift Note**: In OpenShift 3.11, the default node-selector for a project does not allow PSO Operator to mount volumes on master and infra nodes. 
-If you want to mount volumes on master and infra nodes OR run pods in the default namespace using volumes mounted by PSO, then modify the install script as follows.<br/>
+If you want to mount volumes on master and infra nodes OR run pods in the default namespace using volumes mounted by PSO, then set `--node-selector` flag to `""` when run the install script as follows.<br/>
```
Comment (sdodsley) [inline operator-k8s-plugin/README.md]: running, not run

Comment (pure-jliao) [reply operator-k8s-plugin/README.md]: changed.

**Pull request merged**